Libs/src/server/errors/middlewares.ts

```
@@ -0,0 +1,16 @@
+import { ErrorRequestHandler } from "express";
+import { throwInternalServerError } from "./errors-generator";
+import { isHttpError } from "./functions";
+
+export const errorCatcherMiddleware: ErrorRequestHandler = (
+  error,
+  req,
+  res,
+  next
+) => {
+  const { message, code } = isHttpError(error)
+    ? error
+    : throwInternalServerError(error.message || "Internal server error");
```

this doesnt seem right, why should you throw the error? I think you meant `generateInternalServerError`